notebooks/rendered/bigquery-basics.md

```diff
@@ -211,7 +211,7 @@ print('Starting job {}'.format(load_job.job_id))
 load_job.result()  # Waits for table load to complete.
 print('Job finished.')
 
-# Retreive the destination table
+# retrieve the destination table
```

Comment:
nit: lint doesn't seem to have caught this but could you capitalize this for consistency?

Reply:
@dandhlee  I just fixed this in a commit because it's a one character change in line with the original author's intent. Feel free to approve and merge when you see fit

Reply:
Sounds good! Thanks Leah!